app/models/query/results.rb

```diff
@@ -209,14 +209,18 @@ def transform_custom_field_keys(custom_field, groups)
   # Returns the columns that need to be included to allow:
   # * sorting
   # * grouping
-  def include_columns
+  def include_columns # rubocop:disable Metrics/AbcSize
```

Comment:
disabling rubocop should not be done. Please restructure the code to get below 15.0

Reply:
Well, I think it is nicely readable code. However, I changed it to something worse that satisfies the rubocop criteria ;-)